src/Appwrite/Platform/Tasks/CalcTierStats.php

```diff
@@ -155,12 +155,12 @@ public function action(Group $pools, Cache $cache, Database $dbForConsole, Regis
                     }
 
                     /** Get Usage stats */
-                    $range = '90d';
+                    $range = '30d';
```

Comment:
Was there a reason this was reduced?

Reply:
Yea this is for the hamster script. Its alright